macos/FluentUI/Colors.swift

```diff
@@ -0,0 +1,304 @@
+				return "gray40"
+			case .gray30:
+				return "gray30"
+			case .gray20:
+				return "gray20"
+			case .gray25:
+				return "gray25"
+			case .gray50:
+				return "gray50"
+			case .gray100:
+				return "gray100"
+			case .gray200:
+				return "gray200"
+			case .gray300:
+				return "gray300"
+			case .gray400:
+				return "gray400"
+			case .gray500:
+				return "gray500"
+			case .gray600:
+				return "gray600"
+			case .gray700:
+				return "gray700"
+			case .gray800:
+				return "gray800"
+			case .gray900:
+				return "gray900"
+			case .gray950:
+				return "gray950"
+			case .communicationBlue:
+				return "communicationBlue"
+			case .communicationBlueTint40:
+				return "communicationBlueTint40"
+			case .communicationBlueTint30:
+				return "communicationBlueTint30"
+			case .communicationBlueTint20:
+				return "communicationBlueTint20"
+			case .communicationBlueTint10:
+				return "communicationBlueTint10"
+			case .communicationBlueShade30:
+				return "communicationBlueShade30"
+			case .communicationBlueShade20:
+				return "communicationBlueShade20"
+			case .communicationBlueShade10:
+				return "communicationBlueShade10"
+			case .dangerPrimary:
+				return "dangerPrimary"
+			case .dangerTint40:
+				return "dangerTint40"
+			case .dangerTint30:
+				return "dangerTint30"
+			case .dangerTint20:
+				return "dangerTint20"
+			case .dangerTint10:
+				return "dangerTint10"
+			case .dangerShade30:
+				return "dangerShade30"
+			case .dangerShade20:
+				return "dangerShade20"
+			case .dangerShade10:
+				return "dangerShade10"
+			case .warningPrimary:
+				return "warningPrimary"
+			case .warningTint40:
+				return "warningTint40"
+			case .warningTint30:
+				return "warningTint30"
+			case .warningTint20:
+				return "warningTint20"
+			case .warningTint10:
+				return "warningTint10"
+			case .warningShade30:
+				return "warningShade30"
+			case .warningShade20:
+				return "warningShade20"
+			case .warningShade10:
+				return "warningShade10"
+			case .successPrimary:
+				return "successPrimary"
+			case .successTint40:
+				return "successTint40"
+			case .successTint30:
+				return "successTint30"
+			case .successTint20:
+				return "successTint20"
+			case .successTint10:
+				return "successTint10"
+			case .successShade30:
+				return "successShade30"
+			case .successShade20:
+				return "successShade20"
+			case .successShade10:
+				return "successShade10"
+			case .presenceAvailable:
+				return "presenceAvailable"
+			case .presenceAway:
+				return "presenceAway"
+			case .presenceBlocked:
+				return "presenceBlocked"
+			case .presenceBusy:
+				return "presenceBusy"
+			case .presenceDnd:
+				return "presenceOffline"
+			case .presenceOffline:
+				return "presenceOffline"
+			case .presenceOof:
+				return "presenceOof"
+			case .presenceUnknown:
+				return "presenceUnknown"
+			}
+		}
+	}
+	
+	// MARK: Physical - grays
+	
+	@objc public static let gray950: NSColor = Palette.gray950.color
```

Comment:
wy do we need the duplication of the palette and these colors. Can we just Palette and have clients call that directly?

Reply:
I think it's for accessibility?

Reply:
Sorry not why do we need this specific color - why do we need to expose both this gray950 and Palette.gray950? Those are the same thing?

Reply:
Talked to @harrieshin and @rdeassis. I learned that storing the color ensures we call the resourceBundle only once, which improves perf especially for colors that are frequently used. However, Rafael is working on moving the color palettes to the YAML file, so I'll remove the constants now to align with his work.

Reply:
Regardless of the move to the yml file, I'd imagine we can ensure good perf and not over-loading the bundle without exposing 2 separate identical API? We can make it an implementation detail of the one API